webpack/components/ProxmoxServer/ProxmoxServerStorage.js

```diff
@@ -0,0 +1,256 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Title, Divider, PageSection, Button } from '@patternfly/react-core';
+import InputField from '../common/FormInputs';
+import ProxmoxComputeSelectors from '../ProxmoxComputeSelectors';
+import HardDisk from './components/HardDisk';
+import CDRom from './components/CDRom';
+import TimesIcon from '@patternfly/react-icons/dist/esm/icons/times-icon';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+const ProxmoxServerStorage = ({
+  storage,
+  storages,
+  volids,
+  from_profile,
+  paramScope,
+  nodeId,
+}) => {
+  const [hardDisks, setHardDisks] = useState([]);
+  const [nextId, setNextId] = useState(0);
+  const [cDRom, setCDRom] = useState(false);
```

Comment:
That is some weird case of camel case with the cDRom ... Not sure how it should be, though. Maybe cdRom or simply cdrom?